isis/src/control/objs/BundleUtilities/IsisBundleObservation.cpp

```diff
@@ -313,15 +301,6 @@ namespace Isis {
    * Initializes the paramater weights for solving
    *
    * @return @b bool Returns true upon successful intialization
-   *
-   * @internal
-   *   @todo Don't like this, don't like this, don't like this, don't like this, don't like this.
-   *         By the way, this seems klunky to me, would like to come up with a better way.
-   *         Also, apriori sigmas are in two places, the BundleObservationSolveSettings AND in the
-   *         the IsisBundleObservation class too - this is unnecessary should only be in the
-   *         BundleObservationSolveSettings. But, they are split into position and pointing.
-   *
-   *   @todo always returns true?
```

Comment:
We actually need this now that we push all of the parameter logic into the observation sub-classes. So, this function now has a reason to exist!